### ISSUE TITLE
Loss trend validation 

### ISSUE THREAD

**Issue by mamrez7** [junyanz/pytorch-CycleGAN-and-pix2pix#1226]
Hello. I hope everyone are doing well. I have trained the CycleGan on set A and set B below:

A--> 
![0_real](https://user-images.githubusercontent.com/25645568/104685291-acf33500-56c8-11eb-9f5f-5d944248572f.png)

B--> 
![093_1_1_real](https://user-images.githubusercontent.com/25645568/104685309-b7153380-56c8-11eb-9f1f-414198f913e5.png)

And the result at epoch 30 is as:

Fake_B -->

![0_fake](https://user-images.githubusercontent.com/25645568/104685388-f2176700-56c8-11eb-815d-feeae0c9981f.png)

Fake_A --> 

![093_1_1_fake](https://user-images.githubusercontent.com/25645568/104685411-fba0cf00-56c8-11eb-853d-2f75dc83337c.png)

My loss trends is as below, **I am wondering if the losses trends look fine to you**? The discriminator losses sit around 0.2, 0.3 in the plot below:
![loss](https://user-images.githubusercontent.com/25645568/104685538-50444a00-56c9-11eb-8766-f70f287bd001.png)
